src/tests/end-to-end/utils/utils.cpp

```diff
@@ -4,6 +4,208 @@
 
 using namespace Antares::Data;
 
+
+void initializeStudy(Study::Ptr study)
+{
+    study->resultWriter = std::make_shared<NullResultWriter>();
+    study->parameters.reset();
+    Data::Study::Current::Set(study);
+}
+
+void setNumberMCyears(Study::Ptr study, unsigned int nbYears)
+{
+    study->parameters.resetPlaylist(nbYears);
+    study->bindingConstraints.resizeAllTimeseriesNumbers(nbYears);
+}
+
+void configureLinkCapacities(AreaLink* link)
+{
+    const double linkCapacityInfinite = +std::numeric_limits<double>::infinity();
+    link->directCapacities.resize(1, 8760);
+    link->directCapacities.fill(linkCapacityInfinite);
+
+    link->indirectCapacities.resize(1, 8760);
+    link->indirectCapacities.fill(linkCapacityInfinite);
+}
```

Comment:
I don't think this is the right way to set infinite capacities. It will mess with Sirius.

You should use set `link->transmissionCapacities = LocalTransmissionCapacities::infinite` instead.